tests/Makefile.am

```diff
@@ -115,6 +116,14 @@ test_update_refspec_checkpoint_LDFLAGS = $(test_ldflags)
 test_update_refspec_checkpoint_LDADD = $(test_ldadd)
 test_update_refspec_checkpoint_SOURCES = test-update-refspec-checkpoint.c
 
+test_update_direct_CPPFLAGS = $(test_cppflags)
+test_update_direct_CFLAGS = $(test_cflags)
+test_update_direct_LDFLAGS = $(test_ldflags)
+test_update_direct_LDADD = $(test_ldadd)
+test_update_direct_SOURCES = test-update-direct.c \
+				$(top_srcdir)/src/eos-updater-generated.c \
+				$(top_srcdir)/src/eos-updater-generated.h
```

Comment:
Should this be s/`top_srcdir`/`top_builddir`?

Reply:
Also, please break lists up like this:
```
test_update_direct_SOURCES = \
	test-update-direct.c \
	$(top_builddir)/src/eos-updater-generated.c \
	$(top_builddir)/src/eos-updater-generated.h \
	$(NULL)
```

That makes the indentation a bit more consistent. The trailing `$(NULL)` reduces diff noise if more files are added to the list in future.